src/analytics.js

```diff
@@ -56,3 +58,24 @@ export function triggerAnalyticsEvent(nodeOrDoc, eventType, opt_vars) {
     analytics.triggerEvent(eventType, opt_vars);
   });
 }
+
+/**
+ * Helper method to create analytics element for specific root.
+ * @param {!Element} element
+ * @param {!JSONType} config
+ * @param {Document=} parentDoc
+ */
+export function insertAnalyticsElement(element, config, parentDoc) {
```

Comment:
I don't think we need `parentDoc` here. We just should use `element.ownerDocument.createElement(...)`